eq-author-api/schema/resolvers/base.js

```diff
@@ -120,7 +129,7 @@ const createNewQuestionnaire = input => {
 
 const Resolvers = {
   Query: {
-    questionnaires: () => getQuestionnaireList(),
+    questionnaires: (root, args, ctx) => getQuestionnaireList(ctx.user),
```

Comment:
Can we define this inline as it is only used here.

Reply:
Defined it inline instead.